jest.config.js

```diff
@@ -54,8 +54,6 @@ module.exports = {
         testsForPackage('plugin-react-native-session')
       ],
       setupFiles: [
-        require.resolve('react-native/Libraries/Core/setUpGlobals.js'),
-        require.resolve('react-native/Libraries/Core/setUpXHR.js'),
```

Comment:
apparently i thought this was necessary for the rn integration tests https://github.com/bugsnag/bugsnag-js/commit/83bbb38ed617cc5d835e4013600c2b9f53832eec#diff-2d0cd5d10b9604941c38c6aac608178a

but since they still pass on this branch, it seems it is not required